modulefiles/gfsutils_noaacloud.intel.lua

```diff
@@ -2,15 +2,15 @@ help([[
 Build environment for GFS utilities on NOAA Cloud
 ]])
 
-prepend_path("MODULEPATH", "/contrib/spack-stack/spack-stack-1.6.0/envs/unified-env/install/modulefiles/Core")
-prepend_path("MODULEPATH", "/contrib/spack-stack/spack-stack-1.6.0/envs/gsi-addon-env/install/modulefiles/Core")
+prepend_path("MODULEPATH", "/contrib/spack-stack-rocky8/spack-stack-1.6.0/envs/gsi-addon-env/install/modulefiles/Core")
+prepend_path("MODULEPATH", "/apps/modules/modulefiles")
+load("gnu")
+load("stack-intel")
+load("stack-intel-oneapi-mpi")
+unload("gnu")
```

Comment:
Can you explain why gnu is being loaded and then unloaded? 
What errors (if any) are seen?
If there are errors, they are likely errors originating from the installation of spack stack and the source of gnu used there.

Reply:
Without load gnu, we got error:

[Wei.Huang@awsepicweihpca96xlarg-10 gfs-utils-cloud]$ ush/build.sh
WARNING: UNKNOWN PLATFORM
Lmod has detected the following error:  Cannot load module "intel/2023.2.0". At least one of these module(s) must
be loaded:
   gnu

While processing the following module(s):
    Module fullname           Module Filename
    ---------------           ---------------
    intel/2023.2.0            /apps/modules/modulefiles/intel/2023.2.0
    stack-intel/2021.10.0     /contrib/spack-stack-rocky8/spack-stack-1.6.0/envs/gsi-addon-env/install/modulefiles/Core/stack-intel/2021.10.0.lua
    gfsutils_noaacloud.intel  /contrib/Wei.Huang/dev/gfs-utils-cloud/modulefiles/gfsutils_noaacloud.intel.lua

Reply:
Why is it returning `WARNING: UNKNOWN PLATFORM`?  It should have detected `noaacloud`, no?